eureka2-core/src/main/java/com/netflix/eureka2/connection/RetryableConnectionFactory.java

```diff
@@ -124,5 +129,23 @@ public void call(CHANNEL channel) {
         );
     }
 
-    protected abstract Observable<Void> executeOnChannel(CHANNEL channel, OP op);
```

Comment:
the abstract executeOnChannel() functionality is now provide as a func1/func2 that are passed in, so RetryableConnectionFactory is no longer abstract.